src/PostgREST/CLI.hs

```diff
@@ -71,6 +83,13 @@ data CLI = CLI
   }
 
 data Command
+  = Admin AdminCommand
+  | Run RunCommand
+
+data AdminCommand
+  = CmdReady
+
+data RunCommand
```

Comment:
Hm, `Admin AdminCommand` is not clear as this is not a command ran by the Admin Server. I think it should be `ClientCommand`.